usr.bin/netstat/nhops.c

```diff
@@ -64,16 +64,15 @@
 #include "common.h"
 
 /* column widths; each followed by one space */
+#define WID_IF_DEFAULT		12	/* width of netif column */
 #ifndef INET6
 #define	WID_DST_DEFAULT(af) 	18	/* width of destination column */
 #define	WID_GW_DEFAULT(af)	18	/* width of gateway column */
-#define	WID_IF_DEFAULT(af)	(Wflag ? 10 : 8) /* width of netif column */
 #else
```

Comment:
I'm wondering if the *wide* mode should have `IFNAMSIZ` which is default to 16 as the width.

Reply:
this is a good idea, i will re-submit the patch to work like this.